### DISCUSSION TITLE
Build with a collaboration feature?

### DISCUSSION THREAD

**Discussion by l3td33r** [nesbox/TIC-80#1760] [Q&A] [answered]
Hi I have a question, has anyone tried to build TIC-80 with #961 ?
I keep trying but to no avail. It could be because I'm using Manjaro and instructions are for other distros, not sure.

I'd be grateful if someone has a spare build to share!

**Comment by nesbox** [comment, marked as answer]
It's very outdated, the PR was added in 2019 and I think we have to wait until @wadetb  adapts it to the current sources.
